docs/sdk-and-tools/sdk-py/installing-mxpy.md

```diff
@@ -5,19 +5,57 @@ title: Installing mxpy
 
 [comment]: # (mx-abstract)
 
-How to install mxpy
-
-This page describes how to install **mxpy** (the CLI tool). The recommended way to install **mxpy** is by using the `mxpy-up` script.
+This page describes how to install **mxpy** (the CLI tool). The recommended way to install **mxpy** is by using **pipx**. If you want to learn more about **pipx** you can check out [this page](https://pipx.pypa.io/stable/#overview-what-is-pipx).
 
 **mxpy** is currently supported on Linux and MacOS. Some of its features might work on Windows as well, although using **mxpy** on Windows is neither recommended, nor supported at this time.
 
 [comment]: # (mx-context-auto)
 
 ## **Prerequisites**
 
-Before installing **mxpy**, please make sure you have a working **Python 3** environment:
+Before installing **mxpy**, please make sure you have a working **Python 3** environment. You'll need **Python 3.8** or later on Linux or MacOS.
+
+[comment]: # (mx-context-auto)
+
+## **Install using pipx**
+
+:::info
+Keep in mind that installing using **pipx** is only available for **mxpy v9.2.0** or later.
+:::
+
+You'll need **pipx** installed on your machine. For more details on how to install **pipx** check out [this page](https://pipx.pypa.io/stable/#install-pipx).
 
-- **3.8** or later on Linux and MacOS
+In order to install **mxpy** using `pipx`, run the following command:
+
+```sh
+pipx install multiversx-sdk-cli
+```
+
+This will simply install the latest version available.
+
+In case you want to install a specific version you should also specify the version.
+```sh
+pipx install multiversx-sdk-cli==9.2.0
+```
+
+You can also install **mxpy** directly from a GitHub branch. Replace `branch_name` with your desired branch and run the following command:
```

Comment:
And this makes me think - do we still need to maintain the PyPI releases for mxpy, if installation from GitHub is so easy? We should brainstorm this a bit.

I am not referring to the Python libraries, just to mxpy.

Reply:
Well, I don't think it's that much work to maintain the PyPI releases. Indeed, the installation from GH it's pretty simple as well, but I think it's much easier to just write `multiversx-sdk-cli` instead of `git+https://github.com/multiversx/...`. We can ask for other opinions too, but I'd keep it as it is.

Reply:
Well, when using it you still do `mxpy contract...`, so the alias stays. It is just when you install it that you have to type `multiversx-sdk-cli` because that's the name of the package.

Reply:
yes, I'm aware of that, which is good at least.

Reply:
The problem is here:
https://pypi.org/project/mxpy/
https://pypi.org/project/multiversx-sdk-cli/
Confusion of the highest orda